File: practice/palindrome.py
"""
Write a function that returns True if the input is a palindrome, or False, if it is not.

>>> palindrome('hannah')
True

>>> palindrome('racecar')
True

>>> palindrome('a man, a plan, a canal, Panama')
True

>>> palindrome("1 pound coconut.")
False

>>> palindrome(1234321)
True

"""

def palindrome(word):

    word = str(word).lower() ##("What word would you like to check if it's a Palindrone?").lower()

    word= str(word).replace(',', '').replace(' ','')

    if word[::-1] == word:
        return True
    else:
        return False

##palindrome(word)

## Finish this, i'm close..!

File: JS Practice/palindrome.js
/**
 * Created by Anders on 2/21/2017.
 */

(function app() {
    "use strict";

    function palindrome(word) {

        var re = /[^A-Za-z0-9]/g;
        word = word.toLowerCase().replace(re, '');
        var len = word.length;
        for (var i = 0; i < len / 2; i++) {
            if (word[i] !== word[len - 1 - i]) {
                alert('False');
            }
        }
        alert('True');

    }

    var word = prompt("What word or number would you like to check if palindrome? >> ");
    palindrome(word);
})();